packages/components/src/components/action-bar/action-bar.scss

```diff
@@ -85,4 +113,32 @@
   @apply justify-end;
 }
 
+:host([scale="s"]) {
+  --calcite-internal-action-bar-gap: var(--calcite-action-bar-items-space, var(--calcite-spacing-xxs));
+  --calcite-internal-action-bar-padding: var(--calcite-spacing-xxs);
+}
+:host([scale="m"]) {
+  --calcite-internal-action-bar-gap: var(--calcite-action-bar-items-space, var(--calcite-spacing-sm));
+  --calcite-internal-action-bar-padding: var(--calcite-spacing-sm);
+}
+:host([scale="l"]) {
+  --calcite-internal-action-bar-gap: var(--calcite-action-bar-items-space, var(--calcite-space-sm-plus));
+  --calcite-internal-action-bar-padding: var(--calcite-spacing-sm-plus);
+}
+
+.container {
```

Comment:
To avoid duplication, please roll these into the existing rule sets. Also applies to `action.scss`.

Reply:
Nice catch!  Weird that I didn't notice that before :)